src/views/List.jsx

```diff
@@ -1,15 +1,24 @@
 import { ListItem } from '../components';
 import { useState } from 'react';
 
-export function List({ data }) {
+export function List({ data, listId }) {
 	const [searchInput, setSearchInput] = useState('');
 
 	const listItemsToDisplay = data.map((item) => {
 		const isItemInSearch = item.name
 			?.toLowerCase()
 			.includes(searchInput.toLowerCase());
 
-		return isItemInSearch ? <ListItem key={item.id} name={item.name} /> : null;
+		return isItemInSearch ? (
+			<ListItem
+				key={item.id}
+				name={item.name}
```

Comment:
I would pass the item here and deconstruct it inside ListItem. 
```<ListItem key={item.id} item={item} />```
   
```
export function ListItem({item}) { 
const {name,itemId,dateLastPurchased,totalPurchases, listId} = item;
```

Reply:
I tried deconstructing the item object as suggested but I couldn't get it to work. I found it easier to read and found that it still functions properly keeping it the same as it is:

`export function ListItem({ 
	name,
	itemId,
	dateLastPurchased,
	totalPurchases,
	listId }) ` 

`<ListItem
				key={item.id}
				name={item.name}
				itemId={item.id}
				dateLastPurchased={item.dateLastPurchased}
				totalPurchases={item.totalPurchases}
				listId={listId}
			/>`

Reply:
@polly89 and I discussed this. We will leave the props as is for now, since this seems like more a matter of code preference rather than a change to performance. If anyone of a performance benefit to destructuring differently, please share!